src/AvtaleSide/steg/BeregningTilskudd/VisUtregningenPanel.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { Context } from '@/AvtaleContext';
+import Ekspanderbartpanel from 'nav-frontend-ekspanderbartpanel';
+import { ReactComponent as StillingProsentIkon } from '@/assets/ikoner/stillingsprosent.svg';
+import { ReactComponent as ManedsLonnIkon } from '@/assets/ikoner/manedsLonn.svg';
+import { ReactComponent as FeriepengerIkon } from '@/assets/ikoner/feriepenger.svg';
+import { ReactComponent as ObligTjenestepensjonIkon } from '@/assets/ikoner/obligTjenestepensjon.svg';
+import { ReactComponent as GraphRefusjonAvLonnIkon } from '@/assets/ikoner/graphRefusjonAvLønn.svg';
+import { ReactComponent as ArbeidsgiveravgiftIkon } from '@/assets/ikoner/arbeidsgiveravgift.svg';
+import './VisUtregningenPanel.less';
+import Undertittel from 'nav-frontend-typografi/lib/undertittel';
+import { Column, Container, Row } from 'nav-frontend-grid';
+import MediaQuery from 'react-responsive';
+
+const VisUtregningenPanel = (props: Context) => {
+    return (
+        <Ekspanderbartpanel border={true} tittel="Vis hele utregningen" apen={true}>
+            <Container fluid={true}>
+                <Row className="visUtregningenPanel__rad">
```

Comment:
Ingen stor sak, men kunne med fordel brukt BEMHelper i denne komponenten for å unngå så mye gjentakelser og gjort det litt mer leselig. Si ifra hvis jeg skal forklare 🙂